openshift-with-appstudio-test/e2e/util.go

```diff
@@ -49,7 +52,7 @@ func dumpPods(ta *testArgs, namespace string) {
 				ta.Logf(fmt.Sprintf("error getting pod logs for container %s: %s", container.Name, err.Error()))
 				continue
 			}
-			b, err := ioutil.ReadAll(readCloser)
+			b, err := io.ReadAll(readCloser)
```

Comment:
ioutil was flagged for some reason then?

Reply:
yes, [deprecated from v1.16](https://pkg.go.dev/io/ioutil)